2023-03-16-dasharo-fidelisguard-z690-first-qubes-certified-desktop.md

```diff
@@ -0,0 +1,61 @@
+| CPU	         | Intel Core i5-12600K, 3.7GHz                                   |
+| Cooling	     | Noctua CPU NH-U12S Redux                                       |
+| RAM	         | Kingston Fury Beast, DDR4, 4x8GB (32 GB Total), 3600 MHz, CL17 |
+| Power Supply | Seasonic Focus PX 750W 80 Plus Platinum                        |
+| Storage      | SSD Intel 670p 512 GB M.2 2280 PCI-E x4 Gen3 NVMe              |
+| Enclosure	   | SilentiumPC Armis AR1                                          |
+
+[![Photo of Dasharo FidelisGuard Z690 with open case](/attachment/posts/dasharo-fidelisguard-z690_2.jpg)](https://3mdeb.com/shop/open-source-hardware/dasharo-fidelisguard-z690-qubes-os-certified/)
+
+This computer comes with a "Dasharo Supporters Entrance Subscription," which includes the following:
+
+- Full access to [Dasharo Tools Suite (DTS)](https://docs.dasharo.com/dasharo-tools-suite/overview/)
+- The latest Dasharo releases issued by the Dasharo Team
+- Special Dasharo updates for supporters
+- Dasharo Premier Support through an invite-only Matrix channel
+- Influence on the Dasharo feature roadmap
+
+[![Photo of Dasharo FidelisGuard Z690 with open case](/attachment/posts/dasharo-fidelisguard-z690_3.jpg)](https://3mdeb.com/shop/open-source-hardware/dasharo-fidelisguard-z690-qubes-os-certified/)
+
+For further details, please see the [Dasharo FidelisGuard Z690](https://3mdeb.com/shop/open-source-hardware/dasharo-fidelisguard-z690-qubes-os-certified/) product page.
+
+[![Photo of the outside of the Dasharo FidelisGuard Z690](/attachment/posts/dasharo-fidelisguard-z690_4.jpg)](https://3mdeb.com/shop/open-source-hardware/dasharo-fidelisguard-z690-qubes-os-certified/)
+
+## Special note regarding the need for `kernel-latest`
+
+Beginning with Qubes OS 4.1.2, the Qubes installer includes the `kernel-latest` package and allows users to select this kernel option from the GRUB menu when booting the installer. At the time of this announcement, `kernel-latest` is **required** for the Dasharo FidelisGuard Z690's graphics drivers to function properly. Therefore, all potential purchasers and users of this model should be aware that they will have to select a non-default option (`Install Qubes OS RX using kernel-latest`) from the GRUB menu when booting the installer. However, since Linux 6.1 has officially been promoted to being a long-term support (LTS) kernel, it will become the default kernel at some point, which means that the need for this non-default selection is only temporary.
+
+## About Dasharo
+
+"Dasharo is an open-source firmware distribution focusing on seamless deployment, clean and simple code, long-term maintenance, professional support, transparent validation, superior documentation, privacy-respecting implementation, liberty for the owners and trustworthiness for all." [Learn more about Dasharo.](https://docs.dasharo.com/osf-trivia-list/dasharo/)
+
+Dasharo is a registered trademark of and a product developed by [3mdeb](https://3mdeb.com/).
```

Comment:
@andrewdavidwong do we have place to mention https://vpub.dasharo.com/e/1/dasharo-user-group-1 ? @marmarek is our VIP guest during vPub and we plan to announce Qubes OS certification during the DUG#1 closing remarks as a gift for community. I assume we will be able to publish before 5PM CET tomorrow.

Reply:
Other than that and @miczyg1 suggestion this post looks very good. Thank you.

Reply:
Since this is a separate event, I think we should announce it separately.

Reply:
@andrewdavidwong if that is possible it would be great.

Reply:
Opened a separate PR: https://github.com/QubesOS/qubes-posts/pull/111

Reply:
Added a link to the event announcement.